spec/lib/mobile_application_platform/security_token/service_spec.rb

```diff
@@ -12,23 +12,70 @@
     let(:log_prefix) { '[MobileApplicationPlatform][SecurityToken][Service]' }
 
     shared_examples 'STS token request' do
-      context 'when an issue occurs with the client request' do
-        let(:expected_error) { Common::Client::Errors::ClientError }
-        let(:expected_error_message) do
-          "#{log_prefix} token failed, client error, status: #{status}, description: #{description}," \
-            " application: #{application}, icn: #{icn}"
+      shared_examples 'Client Errors' do
```

Comment:
This is a nitpick, so feel free to ignore, but the shared examples need to have a more descriptive name in my opinion. Something that reads like natural language from the spec output, like "a request with client errors".